backend/project/settings.py

```diff
@@ -170,3 +195,17 @@
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
 SITE_ID = 1
+
+# Need to change before production
+CORS_ORIGIN_ALLOW_ALL = True
```

Comment:
Looks like you forgot something...

Reply:
@ckirkwood94 did you make this change when we were pair programming yesterday? i didnt touch backend, only front end

Reply:
Did this ever get answered?